build.sh

```diff
@@ -83,18 +97,19 @@ build_coreclr()
         esac
     fi
 
-    export __GeneratedIntermediatesDirPresent="$__IntermediatesDir/Generated" #do not use this variable, it is used below to support incremental build
-    python -c "import sys;sys.path.insert(0,\"$__ProjectRoot/src/scripts\"); from Utilities import *;UpdateDirectory(\"$__GeneratedIntermediatesDirPresent\",\"$__GeneratedIntermediatesDir\")"
+    echo "Cleaning the temp folder of dynamically generated Event Logging files"
+    python -B -Wall -Werror -c "import sys;sys.path.insert(0,\"$__ProjectRoot/src/scripts\"); from Utilities import *;UpdateDirectory(\"$__GeneratedIntermediate/eventprovider\",\"$__GeneratedIntermediateEventProvider\")"
     if  [[ $? != 0 ]]; then
         exit
     fi
 
+    rm -rf "$__GeneratedIntermediateEventProvider"
+
     # Do not do any more processing from now on:
     # All set to commence the build
 
-    echo "Commencing build of native components for $__BuildOS.$__BuildArch.$__BuildType in $__IntermediatesDir"
+    echo "Commencing build of native components for $__BuildOS.$__BuildArch.$__BuildType in $__GeneratedIntermediate"
```

Comment:
It seems this should have stayed unchanged - the components are still built in the __IntermediatesDir, right? Unless I am mistaken, only the python generated stuff is in the __GeneratedIntermediate.